doc/api/sqlite.md

```diff
@@ -523,6 +523,27 @@ exception.
 | `TEXT`    | {string}             |
 | `BLOB`    | {Uint8Array}         |
 
+## `sqlite.backup(sourceDb, destination[, options])`
+
+<!-- YAML
+added: REPLACEME
+-->
+
+* `sourceDb` {DatabaseSync} The database to backup. The source database must be open.
+* `destination` {string} The path where the backup will be created. If the file already exists, the contents will be
```

Comment:
Also not something that needs to be done in this PR but would it make sense to also support passing a `stream.Writable` / `WritableStream` as the destination to allow backups to be directed somewhere other than the filesystem?

Reply:
I don't think that's possible. Backups should be directed to a connection handle (sqlite *). If I'm not wrong is either a file path or an in-memory db (e.g. `:memory:`)